src/styles/blocks/loops.scss

```diff
@@ -0,0 +1,22 @@
+  @for $index from 1 through 31 {
+    .calendar__number:nth-child(#{$index})::before {
+      content: "#{$index}";
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+
+  $days-of-week: (mon: 0, tue: 104px, wed: 204px, thu: 304px, fri: 404px, sat: 504px, sun: 604px);
```

Comment:
what are these magic numbers? how did you calculate them?
please try not to hardcode values